### PR TITLE
ddl: fix ddl update join table.

### PR DESCRIPTION
**Pull request by coocood**
Fix the case when update multiple tables and one table has a column in write-only state.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala @hanfei19910905 PTAL

File: ddl/ddl_db_test.go [hunk 1]
```diff
@@ -494,3 +497,63 @@ func match(c *C, row []interface{}, expected ...interface{}) {
...
+	// Add a new column in write only state.
+	newColumn := &model.ColumnInfo{
+		ID:           100,
+		Name:         model.NewCIStr("c3"),
+		Offset:       2,
+		DefaultValue: 9,
+		FieldType:    *types.NewFieldType(mysql.TypeLonglong),
+		State:        model.StateWriteOnly,
+	}
+	t1Tbl.Cols()
```
Comment (zimulala) [inline ddl/ddl_db_test.go]: Remove the useless line.

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**